README.md

```diff
@@ -53,14 +53,23 @@ Die angeforderten Daten werden ebenfalls als JSON übermittelt.
 
 ## Authentifizierung
 
-Für jeden Request ist eine Authentifizierung erforderlich. Die Authentifizierung erfolgt über einen HTTP Header.
+Für jeden Request ist eine Authentifizierung erforderlich. Die Authentifizierung erfolgt über den OAuth 2.0 Client-Credentials Flow. 
 
-| Request Header Name | Beschreibung                                        |
-|---------------------|-----------------------------------------------------|
-| X-Authentication    | API JWT der Vertriebsorganisation oder des Partners |
+| Request Header Name | Beschreibung           |
+|---------------------|------------------------|
+| Authorization       | OAuth 2.0 Bearer Token |
 
 
-Das API JWT (JSON Web Token) erhalten Sie von Ihrem Ansprechpartner im KreditSmart-Team. Schlägt die Authentifizierung fehl, erhält der Aufrufer eine HTTP Response mit Statuscode **401 UNAUTHORIZED**.
+Das Bearer Token kann über die [Authorization-API](https://github.com/europace/authorization-api) angefordert werden. 
```

Comment:
In der verlinkten API ist diese hier noch nicht erwähnt. Sollten wir nicht vergessen 🙂

Reply:
https://github.com/europace/authorization-api/pull/3